app/controllers/helpers/LinkWithHiddenValueHelper.scala

```diff
@@ -14,17 +14,12 @@
  * limitations under the License.
  */
 
-package models.requests
+package controllers.helpers
 
-object ExportsSessionKeys {
+import play.api.mvc.{AnyContent, Request}
 
-  val declarationId = "declarationId"
-  val declarationType = "declarationType"
+object LinkWithHiddenValueHelper {
```

Comment:
Yeah, post our conversation after stand up, lets drop this idea and stick to showing the value in the link :+1: